### DISCUSSION TITLE
screenshot in inline mode

### DISCUSSION THREAD

**Discussion by k3it** [n3d1117/chatgpt-telegram-bot#413] [Ideas]
I wonder what we can do to make the screenshot function work in inline mode.  it currently responds with 

`unavailable_message = localized_text("function_unavailable_in_inline_mode", bot_language)`

I really like how it augments the telegram's own URL preview.   Would be nice if it worked inline.

![image](https://github.com/n3d1117/chatgpt-telegram-bot/assets/9923389/b82f64b3-ce40-494a-a12c-b5db40cd95a6)

**Comment by n3d1117** [comment]
I'd love this too, moving to discussions
